seqr/views/utils/orm_to_json_utils_tests.py

```diff
@@ -107,23 +107,6 @@ def test_json_for_igv_sample(self):
 
         self.assertSetEqual(set(json.keys()), IGV_SAMPLE_FIELDS)
 
-    def test_json_for_saved_variant(self):
```

Comment:
I egenrally do not like testing helper methods directly because its very hard to maintain the tests. In this case, the behavior of this helper is now backend specific, which is hard to test here. I confirmed that all the edge cases covered by this test are covered by other tests elsewhere